draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -303,236 +425,233 @@ MUST provide the bit-security required to source input randomness for PQ/T
 components from a seed that is expanded to a output length, of which a subset
 is passed to the component key generation algorithms.
 
-## Key Derivation Function `KDF` {#kdf}
+# Hybrid KEM Schemes {#schemes}
+
+In this section, we define three schemes for building for hybrid KEMs:
+
+* HashEverything - A generic construction that is suitable for use with any choice
+  of traditional and PQ KEMs, with minimal security assumptions on the
+  constituent KEMs
+* PreHashedKeys - A performance optimization of HashEverything for the case
+  where encapsulation keys are large and frequently reused
+* HashTraditionalOnly - An optimized generic construction for the case where the traditional
+  component is a nominal group and the PQ component has strong binding
+  properties
```

Comment:
Concrete suggestions:

GHP - named after GHP2018 which is the foundation of all other combiners basically
PRE - prehash
QSF - same as QSF in X-Wing

Short, mostly meaningful, done.

```suggestion
* GHP - A generic construction that is suitable for use with any choice
  of traditional and PQ KEMs, with minimal security assumptions on the
  constituent KEMs
* PRE - A performance optimization of HashEverything for the case
  where encapsulation keys are large and frequently reused
* QSF - An optimized generic construction for the case where the traditional
  component is a nominal group and the PQ component has strong binding
  properties
```

@bifurcation @br-hale @bwesterb @pag-crypto